config.go

```diff
@@ -175,15 +179,30 @@ func stringsToStyle(raw []string) *Style {
 		bg: termbox.ColorDefault,
 	}
 
+	reFg := regexp.MustCompile("^\\d{1,3}$")
+	reBg := regexp.MustCompile("^on_(\\d{1,3})$")
+
 	for _, s := range raw {
 		fg, ok := stringToFg[s]
 		if ok {
 			style.fg = fg
+		} else {
+			if matchFg := reFg.FindString(s); matchFg != "" {
```

Comment:
This regexp match seems unnecessary. Just use ParseUint, and if it works, it works.
You also might want to check if Use256Colors is enabled before trying this anyway.